EN/00_Applicable-Scope.md

```diff
@@ -1,26 +1,26 @@
-## Cloud Usage Profiles 
+## Cloud Usage Profiles
 
 A summary of the different types of cloud usage is outlined in the description of the profile in the table below:
 
-| Ref # | Profile | Characteristics | Applicable Service Model | Connection Type | 
+| Ref # | Profile | Characteristics | Applicable Service Model | Connection Type |
 |- |- |- |- |- |
-| 1 | Experimentation/Sandbox | <ul><li>Cloud-based services used for experimentation/sandbox</li><li>No direct system to system network interconnections required with GC data centers </li></ul>	| IaaS, PaaS, SaaS | Type 1 - EIS/IIS |
-| 2 | Non-sensitive cloud-based services | <ul><li>Cloud-based services hosting non-sensitive GC content</li><li>No direct system to system network interconnections required with GC data centers </li></ul>| IaaS, PaaS, SaaS | Type 1 - EIS/IIS |
-| 3 | Sensitive (up to PB) cloud-based services | <ul><li>Cloud-based services hosting sensitive (up to Protected B) information</li><li>No direct system to system network interconnections required with GC data centers</li></ul> | IaaS, PaaS, SaaS | Type 1 - EIS/IIS |
-| 4 | Sensitive (up to PB) cloud-based services for GC-wide SaaS Solutions | <ul><li>Cloud-based services hosting sensitive (up to Protected B) information for GC-wide enterprise applications (SaaS)</li><li>No direct system to system network interconnections required with GC data centers </li></ul>| SaaS | Type 2 - IXP |
-| 5 | GC to GC only (Hybrid IT - extension of GC Data Centers) | <ul><li>Hybrid IT environment with an extension of GC network to cloud-based virtual private cloud (up to Protected B) information</li><li>GC cloud-based systems required to interact with systems in GC data centers</li><li>Restricted environment to GC users only</li><li>No external user connections to/from GC cloud-based virtual private cloud and no publicly accessible services</li></ul> | IaaS, PaaS | Type 3 - CXP |
-| 6 | Cloud-based services with External user access and interconnection to GC data centers | <ul><li>Cloud-based services hosting sensitive (up to Protected B) information</li><li>GC cloud-based systems required to interact with systems in GC data centers</li><li>Environment accessible for both GC users and External users and services</li><li>Solution implemented, managed and operated by a GC department/agency</li></ul> | IaaS, PaaS | Type 3 - CXP |
+| 1 | Experimentation/Sandbox |{::nomarkdown}<ul><li>Cloud-based services used for experimentation/sandbox</li><li>No direct system to system network interconnections required with GC data centers </li></ul>{:/}| IaaS, PaaS, SaaS | Type 1 - EIS/IIS |
```

Comment:
Hadn't seen that before, but I guess it's a weird Kramdown limitation that I didn't see any other workarounds when looking around. Too bad it looks weird on the GitHub UI, but right on the site